clang/lib/StaticAnalyzer/Checkers/StreamChecker.cpp

```diff
@@ -268,8 +268,12 @@ class StreamChecker : public Checker<check::PreCall, eval::Call,
         std::bind(&StreamChecker::evalUngetc, _1, _2, _3, _4), 1}},
       {{{"fseek"}, 3},
```

Comment:
```suggestion
      {{{"fseek", "fseeko"}, 3},
```
The same is applicable to `fopen` and `fdopen` in `CallDescriptionMap`.

Reply:
This is not fixed yet, `fseek` and `fseeko` (and `ftell`, `ftello`) should be movable to one entry.